src/sources/soundsourceproxy.cpp

```diff
@@ -177,6 +180,13 @@ QList<QMimeType> mimeTypesForFileType(const QString& fileType) {
             }
         }
     }
+    if (fileType == "stem.mp4" || fileType == "stem.m4a") {
+        // *.stem.mp4 and *.stem.m4a suffix do not have a specific MIME type associated
+        // with them, and simply fall back to MP4. To prevent conflicts with the
+        // MP4 decoder able to decode arbitrary MP4 file (such as video,
+        // extracting just the audio track), we don't return any MIME here
```

Comment:
I think they have. At least libmagic (via Qt) will return one, I guess

Reply:
Ok, so please adjust the comments accordingly. 
Maybe this will receive some refactoring when implementing the stem lookup. 

Another (crazy) thought:
I can imagine to put two or more aac streams into an MP4. How to deal with them, play only the first stream like we do now or treat them like stem files?

Reply:
OK, sounds reasonable.